File: Lab1/Lab1.py
from math import sqrt
x1, y1 = map(float, input( 'Введіть координати першої точки: ' ).split())
x2, y2 = map(float, input( 'Введіть координати другої точки: ' ).split())
result = sqrt( (x2 - x1)**2 + (y2 - y1)**2 )
print( 'Відстань між точками - ', result )

File: Lab1/Lab1.cpp
#include <iostream>
#include <cmath>

int main()
{
    float x1, x2, y1, y2;
    std::cout << "Введіть координати першої точки: ";
    std::cin >> x1 >> y1;
    std::cout << "Введіть координати другої точки: ";
    std::cin >> x2 >> y2;
    float result = sqrt((x1 - x2) * (x1 - x2) + (y1 - y2) * (y1 - y2));
    std::cout << "Відстань між точками - " << result << std::endl;
    return 0;
}